src/components/grid-view/GridView.scss

```diff
@@ -5,29 +5,65 @@
     padding: 0 5px;
 
     &.bdl-GridView--columns-1 .bdl-GridView-slot {
-        $columns1prct: 100%;
+        $columns1prct: calc(100% - 10px);
 
         flex-basis: $columns1prct;
         max-width: $columns1prct;
     }
 
+    &.bdl-GridView--columns-2 .bdl-GridView-slot {
```

Comment:
Can we use the `columns` [style rules](https://developer.mozilla.org/en-US/docs/Web/CSS/columns) to eliminate all this math? It appears to be supported in IE11.

Reply:
Was able to remove this classes and put the width in the javascript.  Still looking into using padding instead of margins.  (Have it working, just a little difficult to get everything perfect.)

Reply:
Have it working using the padding approach.

Reply:
<img width="1626" alt="Screen Shot 2019-07-29 at 10 34 09 AM" src="https://user-images.githubusercontent.com/43506937/62069119-6e27a180-b1ec-11e9-84fb-972a8f2e18d9.png">